pkg/loop/internal/median.go

```diff
@@ -53,6 +53,18 @@ func (m *PluginMedianClient) NewMedianFactory(ctx context.Context, provider type
 		}
 		deps.Add(juelsPerFeeCoinDataSourceRes)
 
+		var gasPriceDataSourceID uint32
```

Comment:
Could this route incorrectly? the ID is 0, so we either need to treat this value differently in the client or signal presence/absence in some other way.